surah/viewsets.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
```

Comment:
Is it a good practice to put views and viewsets into separate files? At least I'm not aware of it :)